bundles/org.eclipse.ui.workbench/Eclipse UI/org/eclipse/ui/internal/themes/ColorsAndFontsPreferencePage.java

```diff
@@ -1304,6 +1320,27 @@ private void updateThemeInfo(IThemeManager manager) {
 			labelProvider.hookListeners(); // rehook the listeners
 	}
 
+	private RGB getColorTakingOverriddenCssValueIntoAccount(ColorDefinition definition) {
+		RGB value = definition.getValue();
+		IPreferenceStore store = getPreferenceStore();
+		if (store != null) {
```

Comment:
instead of increasing nesting: can we invert the condition and do an early "return value;" to increase readability?